Zend/zend_language_scanner.l

```diff
@@ -2321,15 +2487,22 @@ double_quotes_scan_done:
 
 heredoc_scan_done:
 	yyleng = YYCURSOR - SCNG(yy_text);
+	ZVAL_STRINGL(zendlval, yytext, yyleng);
```

Comment:
I think it would be preferable to keep using `yyleng - newline` here and save the `newline` parameter to `strip_multiline_string_indentation`.

Reply:
This is probably not a good idea to do because indented closing markers will have a new line, and then some whitespace after that. If we just take away `newline` from `yyleng` here, it will remove 1 or 2 whitespaces from the end of yytext (or 1 whitespace and `\n`, leaving potentially just `\r` on Windows). This will cause a number of complications for `strip_multiline_string_indentation`, so leaving it as-is would be best.